src/foam/u2/wizard/data/FacadeWizardletSaver.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ foam.CLASS({
+  package: 'foam.u2.wizard.data',
+  name: 'FacadeWizardletSaver',
+  extends: 'foam.u2.wizard.data.ProxySaver',
+
+  imports: [
+    'wizardlets'
+  ],
+
+  properties: [
+    {
+      class: 'FObjectArray',
+      name: 'facadeWizardletSpecs',
+      of: 'foam.u2.wizard.wizardlet.FacadeWizardletSpec',
+      factory: function(){
+        return [];
+      }
+    }
+  ],
+  
+  methods: [
+    async function save(data) {
```

Comment:
Does this need async?

Reply:
Saver interface has async: true for the save method so I just always add it in to be consistent:

https://github.com/kgrgreer/foam3/blob/development/src/foam/u2/wizard/data/Saver.js#L16